.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+      "no-unused-vars": "off",
+      "consistent-return": "off",
+      "func-names": "off",
+      "no-console": "off"
+    }
+}
```

Comment:
Please use same config provided by Microverse for all js projects and re-run linter error checks [ [config here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) ]